runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -239,31 +237,33 @@ public IClientRouter addClient(IClient client) {
      * @return The first client that matches that type.
      * @throws NoSuchElementException If there are no clients matching that type.
      */
-    @SuppressWarnings("unchecked")
     public <T extends IClient> T getClient(Class<T> clientType) {
-        return (T) clientList.stream()
-            .filter(clientType::isInstance)
-            .findFirst().get();
+        return clientList.stream()
```

Comment:
Just remove this whole method, its not being used.

Reply:
done